packages/terra-form-select/src/terra-dev-site/test/form-select/EmptySelectField.test.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import SelectField from 'terra-form-select/lib/SelectField';
+
+const SelectFieldExample = () => (
+  <SelectField label="T-shirt size" placeholder="Select a size" selectId="tshirt-size-field-2" required />
```

Comment:
I hate to be picky, but can we re-use an existing example like: https://engineering.cerner.com/terra-core/tests/terra-form-select/form-select/uncontrolled-search

we should be able to enter a bad value search result to generate "no results found"

Reply:
Reused test in ef3f17d